packages/build-info/src/frameworks/framework.ts

```diff
@@ -39,6 +39,13 @@ export type Detection = {
 
 export type FrameworkInfo = ReturnType<Framework['toJSON']>
 
+export type BuildPlugin = {
+  name: string
+  /** Plugins that should be always installed */
+  alwaysInstall?: boolean
```

Comment:
Can you add a bit more context to this property? When will it be used? I don't see it being pulled in yet?

Reply:
The reference is in the PR description

Reply:
Currently, we have a lot of hardcoded instances of the next runtime inside the react UI and the CLI, where we determine if it needs to be auto-installed:

https://github.com/netlify/netlify-react-ui/blob/main/apps/netlify-react-ui/src/actions/repos.ts#L219-L239
https://github.com/netlify/cli/blob/main/src/utils/init/utils.ts#L19-L43